lib/msal-browser/src/interaction_handler/RedirectHandler.ts

```diff
@@ -45,7 +45,7 @@ export class RedirectHandler extends InteractionHandler {
      * Handle authorization code response in the window.
      * @param hash
      */
-    async handleCodeResponse(locationHash: string, browserCrypto?: ICrypto): Promise<AuthenticationResult> {
+    async handleCodeResponse(locationHash: string, browserCrypto?: ICrypto, clientId?: string): Promise<AuthenticationResult> {
```

Comment:
Should this just be a required parameter?

Reply:
I don't think it can be since handleCodeResponse overrides the handleCodeResponse in InteractionHandler which does not specify clientId as a required parameter

Reply:
We could technically just rename the function here I think and be able to do that, though

Reply:
Ahh that's right. No worries then